sympy/geometry/polygon.py

```diff
@@ -374,6 +374,62 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
+
+    def second_moment_of_area(self, p=None):
```

Comment:
Having default as `p='centroid'` seems more understandable, instead of `None`. And hence update the docstring as well.